shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -32,22 +32,23 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.TreeMap;
 
 /**
  * Prints Alluxio capacity information.
  */
 public class CapacityCommand {
   private static final int INDENT_SIZE = 4;
-  private static final String WORKER_INFO_FORMAT = "%-16s %-16s %-13s %-16s %-13s %-13s %-13s%n";
 
   private BlockMasterClient mBlockMasterClient;
   private PrintStream mPrintStream;
-  private StringBuilder mStringBuilder;
-  private int mIndentationLevel = 0;
-  private long mSumCapacityBytes = 0;
-  private long mSumUsedBytes = 0;
-  private Map<String, Long> mSumCapacityBytesOnTierMap = new HashMap<>();
-  private Map<String, Long> mSumUsedBytesOnTierMap = new HashMap<>();
+  private int mIndentationLevel;
+  private long mSumCapacityBytes;
+  private long mSumUsedBytes;
+  private Map<String, Long> mSumCapacityBytesOnTierMap;
+  private Map<String, Long> mSumUsedBytesOnTierMap;
+  private TreeMap<String, Map<String, String>> mCapacityTierInfoMap;
```

Comment:
`Map<String, Map<String, String>>`? typically we don't specify which map/set/list type on the var declaration, but only on the `new` part when creating new instances.

Reply:
@apc999 When the workers together only have one tier which means the mCapacityTierInfoMap only have one key value pair, I want to use map.firstKey() to get the tierName. map.firstKey() method requires the map type to be TreeMap.

```java
private void printShortWorkerInfo(List<WorkerInfo> workerInfoList) {
    String workerInfoFormat = "%-16s %-16s %-13s %s";
    String tier = mCapacityTierInfoMap.firstKey();
```